.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```diff
@@ -0,0 +1,88 @@
+name: Bug report
+description: Report a bug in pylint
+labels: [bug, needs triage]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please make sure that this issue wasn't already requested, or already implemented in the main branch.
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What is the bug about ?
+      description:
+        Please provide the code you linted, and configurations files and multiple named
+        file if required
+      value: |
+        Given a file `a.py`:
+
+        ```python
+        # Please disable message unrelated to the bug
+        # pylint: disable=missing-docstring,
+        <a> = b + 1
+        ```
+
+        Using the following configuration pylintrc:
+        ```ini
+        # Please remove configuration unrelated to the bug
+
+        [MESSAGES CONTROL]
+        enable=use-symbolic-message-instead,
+        ```
+    validations:
+      required: true
+  - type: textarea
+    id: current-behavior
+    attributes:
+      label: What is the command used and its actual output ?
+      description: Please provide the command used and the current output
+      value: |
+        Result of `pylint a.py`:
```

Comment:
This line is also not really necessary

Reply:
Very often the reporter do not gives the pylint output and just gives a general description "I don't like the output", I'll also keep this, that as it nudges reporter to make a report that permits to understand the problem without thinking too much or linting the code ourselves.